ImmersiveEquipmentMeshGen/MeshHandler.cs

```diff
@@ -11,6 +11,10 @@
 using nifly;
 using Mutagen.Bethesda.Plugins;
 using Mutagen.Bethesda.Archives;
+using IniParser;
```

Comment:
Won't this new dependency break user runs? Not sure how Nuget dependency management works outside of Visual Studio tbh. Seems like _IniParser_ would need to get rolled into the patcher download now.

Reply:
Mutagen uses it too (https://github.com/Mutagen-Modding/Mutagen/blob/b840d5248236559666072f0e71681c713b9a181b/Mutagen.Bethesda.Core/Archives/DI/GetArchiveIniListings.cs#L34), I'm guessing it shouldn't be an issue?